src/libraries/System.Net.Security/src/System/Net/Security/SslStream.cs

```diff
@@ -742,12 +762,33 @@ public override int ReadByte()
             // Otherwise, fall back to reading a byte via Read, the same way Stream.ReadByte does.
             // This allocation is unfortunate but should be relatively rare, as it'll only occur once
             // per buffer fill internally by Read.
-            byte[] oneByte = new byte[1];
-            int bytesRead = Read(oneByte, 0, 1);
+            Span<byte> oneByte = stackalloc byte[1];
```

Comment:
This doesn't need a stackalloc. It can use a local byte and span's ctor, eg https://github.com/dotnet/runtime/blob/6b25c3ba1a349b22e70c5418b5b05e545e1e2271/src/libraries/System.IO.Compression.Brotli/src/System/IO/Compression/dec/BrotliStream.Decompress.cs#L46-L48